docs/low-code-sql/data-tests/data-tests.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Data tests
+id: data-tests
+description: Data tests check the validity of your SQL
+sidebar_position: 8
+tags:
+  - concept
+  - testing
+  - sql
+---
+
+You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project.
+
```

Comment:
I suggest adding some broader info about tests before getting to the specific example below. How about this: 

"Data tests are assertions you make about the tables in your project. 

Data Tests are SQL queries. In particular, they are select statements that seek to grab "failing" records, ones that disprove your assertion. If the data test returns zero failing rows, it passes, and your assertion has been validated."

Reply:
Done.